packages/ember-flight-icons/src/components/flight-icon.js

```diff
@@ -16,7 +16,7 @@ export default class FlightIconComponent extends Component {
       assert('Please provide to <FlightIcon> a value for @name');
     } else if (!iconNames.includes(this.args.name)) {
       assert(
-        `The icon @name "${this.args.name}" provided to <FlightIcon> is not correct. Please verify it exists on https://helios.hashicorp.design/icons/library`
+        `The icon @name "${this.args.name}" provided to <FlightIcon> is not correct. Please verify it exists on https://helios.hashicorp.design/icons/library`,
```

Comment:
I've left a comment in the other PR about this change: https://github.com/hashicorp/design-system/pull/2126#discussion_r1625652230